content/en/tracing/trace_collection/automatic_instrumentation/otel/java.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Using OpenTelemetry Instrumentations with Java
+kind: documentation
+code_lang: java
+type: multi-code-lang
+code_lang_weight: 0
+further_reading:
+    - link: "/opentelemetry/interoperability/"
+      tag: "Documentation"
+      text: "Interoperability with Datadog"
+---
+## Overview
+
+Datadog SDKs support adding instrumentations from OpenTelemetry alongside their built-in instrumentations.
+
+## Compatibility requirements
+
+Support for adding OpenTelemetry instrumentations at runtime was introduced in v1.35.0 of Datadog's SDK for Java.
+
+Each instrumentation should be packaged as an OpenTelemetry [extension](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/examples/extension/README.md).
+The SDK also accepts selected individual instrumentation jars produced by OpenTelemetry's [opentelemetry-java-instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main) build,
+for example the [CFX instrumentation jar](https://search.maven.org/search?q=a:opentelemetry-javaagent-jaxws-2.0-cxf-3.0).
+
+**Note**: Use of OpenTelemetry incubator APIs is not currently supported.
+
+## Getting started
+
+To begin using an OpenTelemetry instrumentation with the Datadog SDK for Java:
+1. Set the `dd.trace.otel.enabled` system property or the `DD_TRACE_OTEL_ENABLED` environment variable to true
+2. Download (or build) the OpenTelemetry instrumentation as an extension jar
+3. Set the `otel.javaagent.extensions` system property or the `OTEL_JAVAAGENT_EXTENSIONS` environment variable to
+   the path to the extension jar
+
+## Configuration
+
+All configuration options below have system property and environment variable equivalents.
+If the same key type is set for both, the system property configuration takes priority.
+System properties can be set as JVM flags.
+
+`dd.trace.otel.enabled`
+: **Environment Variable**: `DD_TRACE_OTEL_ENABLED`<br>
+**Default**: `false`<br>
+Must be set to `true` to enable use of OpenTelemetry instrumentations.
+
+`otel.javaagent.extensions`
+: **Environment Variable**: `OTEL_JAVAAGENT_EXTENSIONS`<br>
+**Default**: `false`<br>
+A comma-separated list of paths to extension jar files, or folders containing extension jar files.
+
+OpenTelemetry's [Agent Configuration](https://opentelemetry.io/docs/zero-code/java/agent/configuration/) page describes additional properties which are also recognized by the Datadog SDK.
+
+## Verified OpenTelemetry Instrumentations
```

Comment:
I'm a bit torn between having this here (one view for all languages) and having it on the existing [language compatibility page](https://docs.datadoghq.com/tracing/trace_collection/compatibility/java?tab=graalvm#networking-framework-compatibility) like we do for Camel

What do you think about having a section on the language-specific compatibility pages and a link from here to those pages?

Reply:
+1 to linking from this OpenTelemetry page to the corresponding language-specific automatic instrumentation page

Reply:
Adding the necessary config detail to the language-specific automatic instrumentation page introduced too much complexity, so what I've done is list the additional Java frameworks that are now covered by adding OpenTelemetry instrumentations there with a link back to here where I can cover how to add them.